### PR TITLE
#61: fix --resume opt

### PR DESCRIPTION
**Pull request by ohDaddyPlease**
Hello. There is fix for https://github.com/Ullaakut/nmap/issues/61
Can you review it and approve?

### CODE REVIEW

File: nmap.go [hunk 1]
```diff
@@ -56,13 +56,25 @@ func NewScanner(options ...func(*Scanner)) (*Scanner, error) {
 
 // Run runs nmap synchronously and returns the result of the scan.
 func (s *Scanner) Run() (result *Run, warnings []string, err error) {
-	var stdout, stderr bytes.Buffer
+	var (
+		stdout, stderr bytes.Buffer
+		f              bool
```
Comment (Ullaakut) [inline nmap.go:61]: Can you name the `f` boolean differently to reflect its actual purpose?

WDYT of `resume` or smth like that?

**Review by Ullaakut — CHANGES_REQUESTED** [review]

File: nmap.go (hunk 1 above)
Comment (ohDaddyPlease) [reply nmap.go:61]: ok! i'll name it `resume `today

**Comment by ohDaddyPlease** [comment]
@Ullaakut ping :)
i have rename it

**Review by Ullaakut — APPROVED** [review]

**Comment by Ullaakut** [comment]
@ohDaddyPlease Thank you for your contribution :)

**Pull request merged**